ios/RCTMGL/RCTMGLMapViewManager.m

```diff
@@ -41,27 +65,28 @@ - (UIView *)view
     mapView.delegate = self;
 
     // setup map gesture recongizers
-    UITapGestureRecognizer *doubleTap = [[UITapGestureRecognizer alloc] initWithTarget:self action:nil];
+    UIShortTapGestureRecognizer *doubleTap = [[UIShortTapGestureRecognizer alloc] initWithTarget:self action:nil];
     doubleTap.numberOfTapsRequired = 2;
-    
-    UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(didTapMap:)];
+
+    UIShortTapGestureRecognizer *tap = [[UIShortTapGestureRecognizer alloc] initWithTarget:self action:@selector(didTapMap:)];
+
     [tap requireGestureRecognizerToFail:doubleTap];
-    
+
```

Comment:
should we set up an editor config that uses their style?

Reply:
Trailing whitespaces aren't really _ever_ a style TBH. I can remove them, but it'll always be a diff-picking job to push up changes.

I think the future for this repo will be that we'll start pushing to it more, so I think we should consider setting it up for our own style. WDYT?